js/scripts/pre-publish.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * publish npm packages
+ */
+import fs from "fs";
+import findUp from "find-up";
+import path from "path";
+import CommandExists from "command-exists"
+import { spawnSync } from "child_process"
+
+
+
+/**
+ * update minor version
+ */
+async function updateVersion(loc: string): Promise<void> {
```

Comment:
I don't you should do this as new version won't be committed. You should commit version bump manually which will trigger release

Reply:
see https://github.com/ChainSafe/filecoindot/runs/4713369165?check_suite_focus=true yarn doesn't work also

Reply:
> I don't you should do this as new version won't be committed. 

agree with this idea, since there are UI packages join into this repo also, and there might be release event in next couple weeks